website/project/decorators.py

```diff
@@ -329,3 +329,20 @@ def wrapped(*args, **kwargs):
 
     # Return decorator
     return wrapper
+
+def must_have_permission_or_public_wiki(func):
+    """ Checks if user has write permission or wiki is public and publicly editable. """
+    @functools.wraps(func)
+    def wrapped(*args, **kwargs):
+        # Ensure `project` and `node` kwargs
+        _inject_nodes(kwargs)
+        node = kwargs['node']
```

Comment:
This is only used once, below - so there's no need to assign it; just use it directly :)